djautotask/sync_rest.py

```diff
@@ -282,9 +282,7 @@ def sync(self):
         results = SyncResults()
         results = self.get(results)
         results.synced_ids = set(
```

Comment:
Can this whole line be removed since synced_ids is already a set that gets IDs added to it?